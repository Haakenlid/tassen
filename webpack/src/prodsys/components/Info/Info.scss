.Info {
  font-size: 2rem;
  font-weight: thin;
  opacity: 0.5;
  flex: 1;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  .icon {
    opacity: 0.5;
    padding: 1rem;
    font-size: 5rem;
  }
}
