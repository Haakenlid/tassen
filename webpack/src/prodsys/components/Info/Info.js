import { Info as Icon } from 'components/Icons'

const Info = ({ children }) => (
  <div className="Info">
    <div className="icon">
      <Icon />
    </div>
    <div className="message">{children}</div>
  </div>
)

export default Info
