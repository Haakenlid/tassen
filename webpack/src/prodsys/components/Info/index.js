import Info from './Info.js'
import './Info.scss'
export default Info
