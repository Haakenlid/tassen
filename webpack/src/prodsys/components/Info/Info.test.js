import Info from './index'
import renderer from 'react-test-renderer'

describe('Info', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Info />).toJSON()
    expect(tree).toEqual(expect.any(Object))
    expect(tree).toMatchSnapshot()
  })
})
