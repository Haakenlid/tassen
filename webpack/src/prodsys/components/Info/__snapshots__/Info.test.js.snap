// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info renders correctly 1`] = `
<div
  className="Info"
>
  <div
    className="icon"
  >
    <svg
      fill="currentColor"
      height="1em"
      preserveAspectRatio="xMidYMid meet"
      style={
        Object {
          "color": undefined,
          "verticalAlign": "middle",
        }
      }
      viewBox="0 0 40 40"
      width="1em"
    >
      <g>
        <path
          d="m21.6 15v-3.4h-3.2v3.4h3.2z m0 13.4v-10h-3.2v10h3.2z m-1.6-25c9.2 0 16.6 7.4 16.6 16.6s-7.4 16.6-16.6 16.6-16.6-7.4-16.6-16.6 7.4-16.6 16.6-16.6z"
        />
      </g>
    </svg>
  </div>
  <div
    className="message"
  />
</div>
`;
